api/src/main/java/io/opentelemetry/api/baggage/BaggageUtils.java

```diff
@@ -5,18 +5,16 @@
 
 package io.opentelemetry.api.baggage;
 
-import com.google.errorprone.annotations.MustBeClosed;
 import io.opentelemetry.context.Context;
 import io.opentelemetry.context.ContextKey;
-import io.opentelemetry.context.Scope;
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.Immutable;
 
 /** Utility methods for accessing the {@link Baggage} contained in the {@link Context}. */
 @Immutable
 public final class BaggageUtils {
```

Comment:
do we still need the class to be public?

Reply:
As long as `TracingContextUtils`, yes, I suggest we keep it public (and `TracingContextUtils` should probably stay public to be Specification compliant).

Reply:
even if none of the methods are public?

Reply:
It does not need to be public, the specification says that the context interaction functionality can be added in few places, and avoid duplicate. So that needs to not be public.

Reply:
You guys can do whatever you want 😉  But if I may give my opinion, it looks really weird to be we have `TracingContextUtils` and no equivalent for `BaggageUtils` (why didn't we have public methods there, btw?)

Also, the specification says:

> they MAY be exposed as static methods on the trace module, as static methods on a class inside the trace module (it MAY be named TracingContextUtilities), or on the Tracer class.

Sure, it only mentions the trace portion, but I would have hoped it would be _similar_ for Baggage ;)